### ISSUE TITLE
New term : "Cardiac hypertrophy"

### ISSUE THREAD

**Issue by AliceChoury** [obophenotype/human-phenotype-ontology#4330]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Cardiac hypertrophy

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

Enlargement of the cardiac muscle tissue. It regroups ventricular hypertrophy, atrial septal hypertrophy, cardiomyocyte hypertrophy,.

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))

Abnormal cardiac ventricle morphology :
[HPO app](https://hpo.jax.org/app/browse/term/HP:0001713)

**Comment by nicolevasilevsky** [comment]
@AliceChoury is this the same as HP_0001639 'Hypertrophic cardiomyopathy'
Definition: Hypertrophic cardiomyopathy (HCM) is defined by the presence of increased ventricular wall thickness or mass in the absence of loading conditions (hypertension, valve disease) sufficient to cause the observed abnormality.

We could add Cardiac hypertrophy as a synonym.

**Comment by LCCarmody** [comment]
Maybe this could be a synonym (Or parent?) of Ventricular hypertrophy HP_0001714 
Def: Enlargement of the cardiac ventricular muscle tissue with increase in the width of the wall of the ventricle and loss of elasticity. Ventricular hypertrophy is clinically differentiated into left and right ventricular hypertrophy.  I do think Hypertrophic cardiomyopathy is distinct. @pnrobinson

**Comment by pnrobinson** [comment]
Let's look into this -- I think the hypertropy is largely ventricular and so this is a near synonym. @AliceChoury  what is the use case for this, i.e., what disease(s) are you thinking of?

**Comment by AliceChoury** [comment]
Thanks for your answer.
Finally, I prefer the term "Hypertrophic cardiomyopathy" (HP:0001639) for my use.

**Comment by nicolevasilevsky** [comment]
Ok great, closing this ticket.
